src/core/package_handling.cpp

```diff
@@ -446,7 +446,7 @@ namespace mamba
                 {
                     if (is_warn || is_fail)
                     {
-                        LOG_WARNING << "Missing file from package cache: " << full_path;
+                        LOG_TRACE << "Missing file from package cache: " << full_path;
```

Comment:
Here I'd keep the warning as it could be a hacker :)

Reply:
same as the other log, 2 warnings are not necessary for the same invalid cache, it's just flooding the logs

Reply:
Again this should almost never happen, and I quite like to get this information when I (myself) changed the files in the package cache, just so that I can figure out bugs that I made myself quicker. In the second warning we're not printing which file has been changed.